stories/page-header.stories.js

```diff
@@ -26,10 +27,13 @@ storiesOf( 'Global', module )
 			<p>
 				<PageHeader
 					title="Module Page Title with Icon"
-					icon
-					iconWidth="23"
-					iconHeight="26"
-					iconID="analytics"
+					icon={
+						<AnalyticsIcon
+							className="googlesitekit-page-header__icon"
+							width={ 23 }
+							height={ 26 }
```

Comment:
Why are these values `23` and `26` while further below they are both `33`? Shouldn't this be square here as well?

Reply:
I'm not 100% sure on the original motivation as this is matching the previous implementation (without, VR tests fail). I will note that that setting values here as `23, 26` would be the same as setting it to `26, 26` as the height is the largest value and SVGs automatically scale the smaller value to fit without stretching (unlike images when setting both).

Reply:
@felixarntz I think it's preferable to keep the dimensions consistent with what were used before for now. I'll open a new issue as a follow up here to cover this, as well as the refactor to provide the icon during module registration.

Reply:
@felixarntz see https://github.com/google/site-kit-wp/issues/2151